test/helper/config.go

```diff
@@ -0,0 +1,38 @@
+package helper
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+)
+
+// NewResourceInterpreterCustomization will build a ResourceInterpreterCustomization object.
+func NewResourceInterpreterCustomization(name string) *configv1alpha1.ResourceInterpreterCustomization {
+	return &configv1alpha1.ResourceInterpreterCustomization{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+		Spec: configv1alpha1.ResourceInterpreterCustomizationSpec{
+			Target: configv1alpha1.CustomizationTarget{
+				APIVersion: "workload.example.io/v1alpha1",
+				Kind:       "Workload",
+			},
+			Customizations: configv1alpha1.CustomizationRules{
+				ReplicaResource: &configv1alpha1.ReplicaResourceRequirement{
+					LuaScript: `
+function GetReplicas(desiredObj)
+  nodeClaim = {}
+  resourceRequest = {}
+  result = {}
+
+  replica = desiredObj.spec.replicas + 1
+  result.resourceRequest = desiredObj.spec.template.spec.containers[1].resources.limits
+  nodeClaim.nodeSelector = desiredObj.spec.template.spec.nodeSelector
+  nodeClaim.tolerations = desiredObj.spec.template.spec.tolerations
+  result.nodeClaim = {}
+  result.nodeClaim = nil
+  return replica, {}
+end`,
+				},
```

Comment:
I think we just test to get replica maybe the lua script can be
```
function GetReplicas(desiredObj)
  replica = desiredObj.spec.replicas + 1
  return replica, {}
end

```

Reply:
Ok. I hope the logic for testing resource requests can be added later.